.eslintrc.js

```diff
@@ -4,9 +4,7 @@ module.exports = {
     jest: true
   },
   rules: {
-    'no-proto': 0,
-    strict: 0,
-    'max-len': 0,
+    'no-proto': 0
```

Comment:
The rule 'no-proto' is set to 0, which effectively turns it off. Ensure this is intentional, as using '__proto__' is generally discouraged due to potential security and performance issues.